src/test/template-analyzer_test.ts

```diff
@@ -0,0 +1,413 @@
+// /* eslint-env mocha */
+import * as assert from 'assert';
+import treeAdapter = require('parse5-htmlparser2-tree-adapter');
+import {TemplateAnalyzer} from '../template-analyzer';
+// @ts-ignore
```

Comment:
😞 what happened

Reply:
i guess you had no types:

```ts
declare module 'babel-eslint' {
  export interface Options {
    requireConfigFile: boolean;
    sourceType: 'module'|'script';
    allowImportExportEverywhere: boolean;
    babelOptions: Record<string, unknown>;
    ecmaFeatures: Partial<{
      globalReturn: boolean;
    }>;
  }
  export function parse(code: string, options?: Partial<Options>);
}
```

in `custom_types/babel-eslint.d.ts`

Reply:
Lol I cheated. Thanks for creating types